src/test/scala/io/qbeast/spark/utils/QbeastCompactionIntegrationTest.scala

```diff
@@ -239,7 +239,7 @@ class QbeastCompactionIntegrationTest extends QbeastIntegrationTestSpec with Log
       val qt = QbeastTable.forPath(spark, tmpDir)
       val m = qt.getIndexMetrics()
       val filePath = m.cubeStatuses.values.flatMap(_.blocks.map(_.file)).head.path
-      qt.optimize(Seq(filePath))
+      qt.optimize(Seq(filePath), Map.empty[String, String])
```

Comment:
Should we leave an empty map as a default value for `options` in the method?

Reply:
I was given the following error: "multiple overloaded alternatives of method optimize define default arguments."

Reply:
I think that is good for a WARN, but we should not overcomplicate the operation for the user...

Reply:
I'll address this in a PR for #337.